<!-- This launch file demonstrates the use of xtion ranger with rtabmap -->
<launch>
	<arg name="map_folder" value="maps" />

	<param name="path_map"  value="$(find ranger_extra)/$(arg map_folder)"/>
	<param name="path_waypoint" value="$(find ranger_extra)/$(arg map_folder)"/>
	<param name="file_map" value="map"/>
	<param name="file_waypoint" value="waypoint.txt"/>
	
	<node pkg="ranger_extra" name="keyboard_talker" type="keyboard_talker.py" output="screen">
	</node>
	<node pkg="ranger_extra" name="mapper" 			type="mapper.py" output="screen">
	</node>
</launch>
